frameworks/kafka/universe/config.json

```diff
@@ -267,6 +267,30 @@
               }
             }
           }
+        },
+        "check": {
+          "description": "To show accurate scheduler plan status in the UI",
```

Comment:
The description is telling what these settings are gonna be immediately _used for_. Maybe it would be better if it actually described what _they are_? Maybe something along the lines of the [health endpoint description](https://github.com/mesosphere/dcos-commons/blob/0.56.0/docs/reference/swagger-api/swagger-spec.yaml#L635)? cc @kaiwalyajoshi

Reply:
@mpereira Thanks for your valuable feedback. Yes, it can be more descriptive.
What if I suggest:
`"description": "Health check to show accurate scheduler plan status in the UI based on the status of the 'deploy' and/or 'recovery' Plans",`

Reply:
That's better. Although I think we can do away with mentioning the UI at all?

```
Health check used to determine the scheduler health based on the status of the 'deploy' and/or 'recovery' plans.
```

Reply:
What do others think? cc @samvantran @zmalik

Reply:
good catch @mpereira 
the recommended message sounds better to me, but maybe just `scheduler plans` instead of `'deploy' and/or 'recovery' plans` as these may include backup/restore plans also and not only deploy and recovery

Reply:
@zmalik agreed.

Reply:
@vibhujain can you update the PR with next value for description?
```
Health check used to determine the scheduler health based on the status of the scheduler plans.
```

Reply:
SGTM

Reply:
I have changed the description as per the suggestion! Thank you @zmalik @mpereira @samvantran !